servers_v7.json

```
@@ -109,7 +109,7 @@
     "address": ["n1.yeet.ml:6577"]
   }
   {
-    "name": "[red]LamNguyen[green]Server
+    "name": "[red]LamNguyen[green]Server"
```

```suggestion
    "name": "[red]LamNguyen[green]Server",
```
My bad, I forgot the comma.